pkg/rootless/mounts.go

```diff
@@ -14,6 +14,16 @@ import (
 )
 
 func setupMounts(stateDir string) error {
+	// Remove symlinks to the rootful files, so that we can create our own files.
```

Comment:
Just to be clear, these are removed from within the child process's overlay filesystem, NOT the root filesystem.

Reply:
Correct. But it is not a real `overlayfs`. It is just a writable tmpfs with symlinks to the real files.